src/reconcile/rec_write.c

```diff
@@ -3516,6 +3516,12 @@ __rec_split_write(WT_SESSION_IMPL *session,
 	 */
 	if (F_ISSET(r, WT_EVICT_LOOKASIDE) && bnd->supd != NULL) {
 		F_SET(dsk, WT_PAGE_LAS_UPDATE);
+		/*
+		 * This is going to mark more pages as LAS than necessary and
+		 * the flag doesn't ever get cleared at the moment, but
+		 * better conservative than wrong.
+		 */
+		r->ref->has_las_records = true;
```

Comment:
I'm not following this: it seems to me (and I'm probably totally wrong), that we have to review LAS records, regardless of their page being in memory or not.